updatecli/updatecli.d/terraform-modules/eks.yml

```diff
@@ -36,14 +36,14 @@ targets:
       matchPattern: >-
         source(.*)=(.*)"terraform-aws-modules/eks/aws"(\r\n|\r|\n)(.*)version(.*)=(.*)
       replacePattern: >-
-        source${1}=${2}"terraform-aws-modules/eks/aws"${3}${4}version${5}= "{{ source `getLatestVersion` }}"
+        source${1}=${2}"terraform-aws-modules/eks/aws"${3}${4}version${5}= "{{ source "getLatestVersion" }}"
```

Comment:
Can you test it please? In this context, YAML consider the string as literal, so the backticks *might* be important